services/QuillLMS/spec/controllers/api/v1/session_feedback_histories_controller_spec.rb

```diff
@@ -0,0 +1,106 @@
+require 'rails_helper'
+
+describe Api::V1::SessionFeedbackHistoriesController, type: :controller do
+  context "index" do
+    it "should return successfully - no history" do
+      get :index
+
+      parsed_response = JSON.parse(response.body)
+
+      assert_response :success
+      assert_equal parsed_response, {
+        total_pages: 0,
+        total_activity_sessions: 0,
+        current_page: 1,
+        activity_sessions: []
+      }.stringify_keys
+    end
+
+    context 'with feedback_history' do
+      it "should return successfully" do
+        create(:feedback_history, entry: 'This is the first entry in history')
+        create(:feedback_history, entry: 'This is the second entry in history')
+
+        get :index
+
+        parsed_response = JSON.parse(response.body)
+
+        assert_response :success
+        assert_equal parsed_response['activity_sessions'].length, 2
+      end
+
+      context 'pagination' do
+        setup do
+          30.times { create(:feedback_history) }
```

Comment:
This is a lot of data to create for a test. Consider [stubbing the constant](https://relishapp.com/rspec/rspec-mocks/v/2-11/docs/stubbing-constants/stub-defined-constant) to something smaller, like 2. and then running this test.

Reply:
Good call.  Fixed it.